src/NumericalAlgorithms/DiscontinuousGalerkin/LiftFlux.hpp

```diff
@@ -42,7 +42,7 @@ auto lift_flux(Variables<tmpl::list<FluxTags...>> flux,
                Scalar<DataVector> magnitude_of_face_normal) noexcept
     -> Variables<tmpl::list<db::remove_tag_prefix<FluxTags>...>> {
   auto lift_factor = std::move(get(magnitude_of_face_normal));
-  // The LGL weights are:
+  // For an Nth order basis, the LGL weights are:
```

Comment:
I've always found the `Nth order basis` naming bad because sometimes the degree of the basis==order and sometimes the degree +1==order. How about `an (N-1)th degree basis` since everyone agrees on what the degree is? Or explicitly say that `N` is the number of basis functions.

Reply:
I guess the fact that this is actually the `N+1`-basis-function case shows how confusing this all is.

I'll go with the extremely explicit "For an Nth degree basis (i.e., one with N+1 basis functions),"

Reply:
Haha, yea, that sounds good :) Thank you!